cronjobs/src/commands/expire_orphan_attachments.py

```diff
@@ -18,10 +19,9 @@
 
 def expire_orphan_attachments(event, context):
     """
-    This cronjob will release holds on orphan attachments.
-    See https://cloud.google.com/storage/docs/holding-objects
-    And then GCS will start the count down, and when the retention
-    period is over, it will soft-delete them.
+    This cronjob will set the current time on orphan attachments.
```

Comment:
Maybe "This cronjob will set the custom time field on orphaned attachments to the current time."?